src/components/Prose.astro

```diff
@@ -0,0 +1,9 @@
+---
+
+---
+
```

Comment:
Leave comment to docs for future devs to look at easier
https://docs.astro.build/en/recipes/tailwind-rendered-markdown/